### DISCUSSION TITLE
How to point to remote directories?

### DISCUSSION THREAD

**Discussion by KillavoltAI** [lllyasviel/stable-diffusion-webui-forge#456] [General]
I have a bit of a custom setup on Ubuntu, in that I have multiple UI's installed, which all point to a single models/venv and other sharable directories, but in Forge this is giving me a bit of an issue.
I have forge installed in a directory called "ForgeUI" instead of the default.

Two problems.

1.) When I uncomment and change the path in webui-user.sh to my venv folder, it doesn't use it and still makes the venv folder in it's install directory. I'm using the same relative path as A1111 "../venv" This works fine in A1111 but doesn't seem to work in Forge.

2.) I can't find the config file to point to my models directory that I also have separately so that all UI's can use the same models. Where do I tell it to look for the model files, styles.csv, and other such "sharable" folders?

**Comment by Guchdorg** [comment]
About the models directory there is some sort of switch when you could run based on the original Automatic1111.  It seem a bit much.   I found it just easier to create a symbolic link in Linux using the "ln" command.  You have to rename or delete whatever directory first.  But the link you create will act like that directory but you can point it somewhere else.

**Reply by KillavoltAI** [reply]
Before I do something like that I'd prefer to find which file points to those directories and change it that way. It must be possible in some file somewhere.

**Comment by KillavoltAI** [comment]
I can get it to use my venv folder by not using a relative path, but now for some reason it keeps cloning sdwebui into a separate folder within /ForgeUI and running that, which  isn't Forge.
